package.json

```diff
@@ -7,9 +7,11 @@
     "build": "next build",
     "dev": "next",
     "deploy": "vercel",
-    "lint": "mastarm lint components pages --quiet"
+    "lint": "mastarm lint components pages --quiet",
+    "predev": "yarn install"
```

Comment:
Is it possible to just say `yarn`?

Reply:
Sure (https://github.com/ibi-group/otp-admin-ui/pull/30/commits/ea940355ef74829804443a392256fca253aeb68d)